releases/7.5.3.md

```diff
@@ -14,6 +14,7 @@ Arches 7.5.3 Release Notes
 - Fixes print from resource manager #11035
 - Displays configured maximum file size and accepted image file formats in photo gallery #11022
 - Alphabetically sort graphs by name in search Resource Type filter #10646
+- Adds check to GeoJSON export that value is a valid geometry # 11139
```

Comment:
If you move this up a few lines, that will resolve the merge conflict. Also, there is a space between the # and the issue number that will prevent github from creating a link.